featurebyte/models/entity_universe.py

```diff
@@ -263,6 +266,20 @@ def get_entity_universe_template(self) -> Expression:
         return universe_expr
 
 
+def get_dummy_entity_universe() -> Select:
+    """
+    Returns a dummy entity universe (actual value not important since it doesn't affect features
+    calculation)
+
+    Returns
+    -------
+    Select
+    """
+    return expressions.select(
+        expressions.alias_(make_literal_value(1), "dummy_entity", quoted=True)
```

Comment:
Do we need to worry name collision here?

Reply:
It should be fine as long as the dummy serving name is consistent.